test/types_integration_test.go

```diff
@@ -20,7 +20,7 @@ func addTypesToTable(db *sql.DB, dbName, tableName string) {
 		"ADD date_col DATE," +
 		"ADD time_col TIME," +
 		"ADD dt_col DATETIME," +
-		"ADD ts_col TIMESTAMP," + // TODO broken on master
```

Comment:
What was this TODO for? Can we add a test for the broken behaviour? I think that was Justin's intention?

Reply:
This was the test that is broken due to this bug if you have a non UTC timezone on your computer/testing setup.

Reply:
Which test? you're just removing a comment here. What am I missing?

Reply:
In other words: the `setupMultiTypeTable` method below is not inserting anything into this column because doing so would have caused the failure you're trying to fix here. But now that we don't have the bug, we should be able to re-insert the value

Reply:
Huh interesting, I recall when debugging distinctly seeing a TZ issue. The test case `TestCopyDataWithManyTypes` also fails without this patch (without modification to setupMultiTypeTable) on my machine with EDT set. 

I'll check this more carefully next week.

Reply:
Need to add the column here https://github.com/Shopify/ghostferry/blob/a37ec446d7b065760d52819903c4b808c8c9e3f3/test/types_integration_test.go#L53

Reply:
So it appears this test was ACTUALLY broken in master (well as it never passed on my machine) because even though we don't INSERT the ts_col explicitly, it's done implicitly. If we run SHOW CREATE TABLE table1, we can see:

```
  `ts_col` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
```

I'll add the ts_col into that query regardless, but that explains why I see the failed test despite never specifying that column.